perun-openapi/openapi.yml

```diff
@@ -6934,7 +7206,35 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
-  #TODO removeSponsorRole
+  /urlinjsonout/vosManager/removeSponsorRole/user:
+    post:
+      tags:
+        - VosManager
+      operationId: removeSponsorRoleFromUser
+      summary: Removes user as a sponsor. His or her sponsored members will be set as expired if the user was their last sponsor.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+        - $ref: '#/components/parameters/userId'
+      responses:
+        '200':
+          $ref: '#/components/responses/VoidResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /urlinjsonout/vosManager/removeSponsorRole/group:
+    post:
+      tags:
+        - VosManager
+      operationId: removeSponsorRoleFromGroup
+      summary: Removes group as a sponsor. All group members will cease to be sponsors, and their sponsored members will be set as expired if the group member was their last sponsor.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+        - { name: authorizedGroup, schema: { type: integer}, in: query, required: true, description: 'Group id' }
```

Comment:
The authorizedGroup parameter is repeated at methods addSponsorRoleToGroup, removeSponsorRoleFromGroup, addVoAdminGroup, removeVoAdminGroup, addFacilityAdminGroup, removeFacilityAdminGroup, addAdminToGroup, removeGroupAdminResource. Please declare it in #/components/parameters and use $ref to reference it.